### PR TITLE
add versioning to readme

### PR DESCRIPTION
**Pull request by tanzimfh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-The mock responses are planned to be used in the Vertex AI for Firebase SDKs in the following repositories:
-- [firebase-android-sdk](https://github.com/firebase/firebase-android-sdk/)
+The mock responses are currently being used to test the Vertex AI for Firebase SDKs in the following repositories:
 - [firebase-ios-sdk](https://github.com/firebase/firebase-ios-sdk/)
-- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
\ No newline at end of file
+- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
+
+## Versioning
+All commits affecting the [mock-responses](https://github.com/FirebaseExtended/vertexai-sdk-test-data/tree/main/mock-responses) directory will be tagged with version numbers of the format `v<MAJOR>.<MINOR>` following these guidelines:
```
Comment (rlazo) [inline README.md:10]: can you format this line so it's not a long single one?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-
-The mock responses are planned to be used in the Vertex AI for Firebase SDKs in the following repositories:
-- [firebase-android-sdk](https://github.com/firebase/firebase-android-sdk/)
+The mock responses are currently being used to test the Vertex AI for Firebase SDKs in the following repositories:
 - [firebase-ios-sdk](https://github.com/firebase/firebase-ios-sdk/)
-- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
\ No newline at end of file
+- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
+
+## Versioning
```
Comment (rlazo) [inline README.md:9]: Is it 'will be tagged' or 'should be tagged'?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+All commits affecting the [mock-responses](https://github.com/FirebaseExtended/vertexai-sdk-test-data/tree/main/mock-responses) directory will be tagged with version numbers of the format `v<MAJOR>.<MINOR>` following these guidelines:
+- Changes to existing mock response files require a **major** version bump.
+- Introduction of new mock response files require a **minor** version bump.
```
Comment (rlazo) [inline README.md:12]: You can also add, that there's no "patch" version at all.

**Review by rlazo — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tanzimfh) [reply README.md:11]: I meant that it will be tagged by GitHub Actions. I added that part in the latest commit.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
\ No newline at end of file
+- [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/)
+
+## Versioning
+All commits affecting the [mock-responses](mock-responses) directory will be
+tagged by GitHub Actions with version numbers of the format `v<MAJOR>.<MINOR>`
+following these guidelines:
+- Changes to existing mock response files require a **major** version bump.
+- Introduction of new mock response files require a **minor** version bump.
+- There is no "patch" version.
```
Comment (rlazo) [inline README.md:17]:
```suggestion
- There is no **patch** version component.
```

**Pull request merged**